examples/maintained/make_prediction_dataflow.py

```diff
@@ -49,30 +47,60 @@
     DNNClassifierModelConfig,
 )
 
+from feature.git.dffml_feature_git.feature.operations import count_authors,git_commits,work,check_if_valid_git_repository_URL
+from feature.git.dffml_feature_git.feature.definitions import *
+
+from dffml.df.base import op
 
-"""
-TODO
-    * Insert prediction to database
-    * make dataflow_yaml general
-    * generalize??
-    * export dataflow -> predict_dataflow
-    * change documentation
-"""
 
 dataflow_yaml = "./cgi-bin/dataflow.yaml"
 
+#operation to collect features [authors,work,commits] to make features to pass to model
+@op(
+    inputs = {
+        "authors" : count_authors.op.outputs["authors"],
+        "work" : work.op.outputs["work"],
+        "commits" : git_commits.op.outputs["commits"]
+        }
+    ,
+    outputs ={
+        "features" : Definition(
+                name="maintained_features",
+                primitive="Dict[str,Any]"
+                )
+            }
+)
+async def collect_maintained_features(authors,work,commits):
+    return {
+        "features":{
+            "authors":authors,
+            "work":work,
+            "commits":commits
+        }
+    }
+
+#does outputing same defintion cause context to run twice?
```

Comment:
Yes. If you output something that's the same type as your input. Unless you update the `flow` to not have the operation accept inputs from itself.